cranelift/README.md

```diff
@@ -85,6 +84,16 @@ Building Cranelift
 Cranelift uses a [conventional Cargo build
 process](https://doc.rust-lang.org/cargo/guide/working-on-an-existing-project.html).
 
+Building all crates in the wasmtime + Cranelift repository will require a
+nightly version of Rust. With stable Rust, one can build the `wasmtime` binary,
```

Comment:
I think we don't have anything at this point that requires nightly, so this may need to be updated?

Reply:
I suggest a small experiment to check your assumption:

1. clean clone
2. cargo test --all with stable rust

I've tried this and it complains about feature(test) being nightly only.

Reply:
Lightbeam uses an unstable feature for benchmarks: https://github.com/bytecodealliance/wasmtime/blob/d2daf5064e5ee2f1c630579e4c6e273a5a504d3c/crates/lightbeam/src/lib.rs#L1

Simply building doesn't use this, only when testing the lightbeam crate is it used.

Reply:
Wasmtime is a big repository and has a lot of projects in it. Mentioning very prominently inthe README that the whole project requires nightly because of one optional dependency only for tests feels a bit disingenuous which is why I think this should be reworded a bit.

Reply:
Fair, I adjusted it to better reflect this notion. Note this the cranelift README not the top level one.